app/models/user.rb

```diff
@@ -0,0 +1,12 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
+  has_many :comments, foreign_key: 'author_id'
+  has_many :likes, foreign_key: 'author_id'
+
+  def latest_posts
+    Post.where(author: self).limit(3)
+  end
+
+  validates :Name, presence: true
+  validates :PostCounter, numericality: { greater_than_or_equal_to: 0 }
```

Comment:
- [ ] [OPTIONAL] I would suggest that you put all your validations on the top of file so that they will be checked before any thing else 😃 👍